### PR TITLE
add support for offline grype db

### PR DESCRIPTION
**Pull request by matthyx**

### CODE REVIEW

File: labs/kubescape-relevancy/values.yaml [hunk 1]
```diff
@@ -296,8 +296,8 @@ kubevuln:
 
   image:
     # -- source code: https://github.com/kubescape/kubevuln
-    repository: quay.io/kubescape/kubevuln
```
Comment (dwertent) [inline labs/kubescape-relevancy/values.yaml:299]: Maybe we should first build the new image

**Review by dwertent — CHANGES_REQUESTED** [review]

File: labs/kubescape-relevancy/values.yaml (hunk 1 above)
Comment (matthyx) [reply labs/kubescape-relevancy/values.yaml:299]: yes, this is for @rcohencyberarmor to run it with system tests

Comment (matthyx) [reply labs/kubescape-relevancy/values.yaml:299]: @dwertent done, and disabled by default

**Comment by dwertent** [comment]
@matthyx can you please bump the helm chart version?

**Comment by matthyx** [comment]
done, sorry

**Pull request merged**